File: frontend/src/types/barber.ts
export interface Barber {
  id: number
  userId: number
  displayName: string
  isActive: boolean
}

export interface CreateBarberRequest {
  name: string
  email: string
  displayName: string
}

export interface UpdateBarberRequest {
  id: number
  displayName: string
}

export interface ChangeBarberStatusRequest {
  id: number
  isActive: boolean
}

File: src/BarberShop.Domain/Barber.cs
namespace BarberShop.Domain
{
    public class Barber : BaseEntity
    {
        public long UserId { get; set; }
        public string DisplayName { get; set; } = string.Empty;
        public bool IsActive { get; set; } = true;

        public User User { get; set; } = null!;
    }
}
